src/billing/components/Free/OrgLimits.tsx

```diff
@@ -104,6 +104,15 @@ export const OrgLimits: FC = () => {
           )
         }
 
+        // The 'dashboards', 'tasks', 'checks', and 'rules' limits are not applicable to orgs using IOx.
```

Comment:
Can we put this check at the beginning of the `return`/`render`, since it looks like it supersedes the other rules about which limits should be displayed?